core/src/main/java/hudson/tools/JDKInstaller.java

```diff
@@ -697,6 +699,9 @@ public JDKRelease getRelease(String productCode) {
     }
 
     public static final class JDKRelease {
+        public JDKFile[] files;
+        public String licpath;
+        public String lictitle;
```

Comment:
Would it be possible to avoid creating new public fields? It increases the technical debt and complicates the class improvement